File: Assignment 8/Problem4.java
//Accept N number from user and print elements which are divisble by 5 and 3


import java.lang.*;
import java.util.*;

class Number
{
	public void Display(int Brr[])
	{
		System.out.println("Elements are :");
		
		for(int iCnt=0;iCnt<Brr.length;iCnt++)
		{
			if((Brr[iCnt]%5)==0 && (Brr[iCnt]%3)==0 )
			{
				System.out.println(Brr[iCnt]);
			}

		}
	}
}
class Problem4
{
	
	public static void main(String arr[])
	{
		Number nobj= new Number();
		
		Scanner sobj = new Scanner(System.in);
		System.out.println("Enter Number of Elements");
		int iSize = sobj.nextInt();
		
		int Arr[]= new int[iSize];
		
		System.out.println("Enter Value of Array : ");
		for(int iCnt=0;iCnt<Arr.length;iCnt++)
		{
			Arr[iCnt]= sobj.nextInt();
		}
		
		nobj.Display(Arr);
	}
}

File: Assignment 11/Problem4.c
//Accept N number from user and Display such number which contain 3 digit in it.

#include<stdio.h>
#include<stdlib.h>

void Digits(int Arr[], int iLength)
{
    int iCnt = 0;
	int iCount=0;
	int iTemp=0;
  
    for(iCnt =0; iCnt<iLength ; iCnt++)
    {	
			iTemp=Arr[iCnt];
			
            while(Arr[iCnt]!=0)
			{
				iCount++;
				
				Arr[iCnt]=Arr[iCnt]/10;
			}
			
			if(iCount==3)
			{
				printf("%d\t",iTemp);
			}
			
			iCount=0;
    }
}

int main()
{
    int iSize = 0, iCnt = 0;
    int *ptr = NULL;   


    printf("Enter number of elements\n");
    scanf("%d",&iSize);

    ptr = (int *)malloc(sizeof(int) * iSize);

    printf("Enter the values\n");
	
    for(iCnt = 0; iCnt < iSize; iCnt++)
    {
		printf("Enter element %d : ",iCnt+1);
        scanf("%d",&ptr[iCnt]);
    }
	
	Digits(ptr, iSize);

	
    return 0;
}